src/charts/stacked-area.js

```diff
@@ -1561,6 +1584,22 @@ define(function(require){
             return this;
         };
 
+        /**
+         * Gets or Sets the yAxisBaseline
```

Comment:
This is the part that goes into API docs. Could you add a more detailed description of what it does here? In addition, providing an example here would be super helpful as well.

Reply:
You can document an example of code like this:
https://github.com/eventbrite/britecharts/blob/master/src/charts/stacked-area.js#L1519

Reply:
Thanks for the hint how to document examples. Will add it.